src/jest-types.ts

```diff
@@ -15,7 +15,20 @@ export type HasteConfig = {
   providesModuleNodeModules: Array<string>;
 };
 
-export type JestConfig = {
+export interface PostProcessorOptions {
+  plugins? : any[];
+  presets?: any[];
+  cacheDirectory? : string;
+  filename?: string;
+}
+
+export interface PostProcessHook {
+	(src: string, filename: string, config: JestConfig, transformOptions: TransformOptions) : string;
```

Comment:
Would a new type to pass into this function be better than multiple params? It'll give a bit of flexibility if we want to change something. For instance, we might want to pass in the original source, the transpiled output and a sourcemap.